stories/components/SearchBar.stories.js

```diff
@@ -0,0 +1,10 @@
+import SearchBar from "../../components/SearchBar";
+
+export default {
+  component: SearchBar,
+  title: "components/Search Bar",
+};
+
+const Template = (args) => <SearchBar {...args} />;
+
+export const Basic = Template.bind({});
```

Comment:
Why `bind` this looks different to our other stories?

Reply:
This is from the storybook docs. We can easily pass arguments and even reuse those arguments.

![image](https://user-images.githubusercontent.com/27003616/222459035-6bb3538f-32da-439a-85bf-8f8a955cf08c.png)


Let's say we have some arguments for `Basic` component. And we need most of the arguments for `Rounded` component the same as well. We can do this

```
export const Basic = Template.bind({});
Basic.args = {
  color: "red",
};

export const Rounded = Template.bind({});
Rounded.args = {
  ...Basic.args,
  rounded: true,
};
```